src/pl/Plajer/GameAPI/LanguageManager.java

```diff
@@ -0,0 +1,79 @@
+package pl.Plajer.GameAPI;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.logging.Level;
+
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public class LanguageManager {
+
+	private static JavaPlugin plugin;
+	private static FileConfiguration languageConfig = null;
+	private static File languageConfigFile = null;
+
+	public LanguageManager(JavaPlugin pl){
```

Comment:
You have all static methods? Either remove static or use a method instead of the constructor for intializing the Javaplugin plugin field. I would all the method "public static void init(Javaplugin plugin);"